sbncode/EventGenerator/PythiaGen/setup.sh

```diff
@@ -0,0 +1,4 @@
+source /cvmfs/icarus.opensciencegrid.org/products/icarus/setup_icarus.sh
+setup pythia8 v8_3_05 -q e20:p392:prof
```

Comment:
This kind of follows an anti-pattern with the respect to other packages. 
Should we not add pythia in the UPS dependencies?

Reply:
I don't think so -- pythia is only needed to run this one notebook and is not needed to build the repository.

I would think it's better to avoid this coupling to an external package unless we need it.